lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -417,6 +417,37 @@ fb_actions.goto_cwd = function(prompt_bufnr)
   current_picker:refresh(finder, { reset_prompt = true, multi = current_picker._multi })
 end
 
+--- Change working directory of nvim to the selected file/folder in |builtin.file_browser|.
+---@param prompt_bufnr number: The prompt bufnr
+fb_actions.change_cwd = function(prompt_bufnr)
+  local picker = action_state.get_current_picker(prompt_bufnr)
+  local finder = picker.finder
+  local sel = fb_utils.get_selected_files(prompt_bufnr, true)[1]
+  finder.path = sel:is_dir() and sel:absolute() or sel:parent():absolute()
+  finder.cwd = finder.path
+  finder.files = true
+  vim.cmd("cd " .. finder.path)
+  if picker.results_border then
+    local title = Path:new(finder.path):make_relative(finder.path) .. os_sep
+    picker.results_border:change_title(title)
+  end
+  picker:refresh(finder, { reset_prompt = true, multi = picker._multi })
+end
+
+--- Goto home directory in |builtin.file_browser|.
+---@param prompt_bufnr number: The prompt bufnr
+fb_actions.goto_home_dir = function(prompt_bufnr)
+  local picker = action_state.get_current_picker(prompt_bufnr)
+  local finder = picker.finder
+  finder.path = vim.fn.expand "~"
```

Comment:
Is this OS-agnostic and supports Windows as well? Nitpick: `vim.env.HOME` from within lua is slightly preferred I suppose.

Reply:
👍